docker-compose.yml

```diff
@@ -25,9 +25,15 @@ services:
       context: ./src/backend
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
-    restart: always
+      - ASPNETCORE_URLS=http://0.0.0.0:8080
+      - SERILOG__WRITETO__1__NAME=Seq
```

Comment:
This adds writing logs to Seq locally. Seq is similar to Splunk. We use Seq only in development to ease viewing, searching of logs.